docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -0,0 +1,100 @@
+\page callback-functions Using Callback Functions
+
+When you create or get an object from Discord, you send the request to its API and in return you get either an error or the object you requested/created. You can pass a function to API calls as the callback function. This means that when the request completes, and you get a response from the API, your callback function executes. You must be careful with lambda captures! Good practice would be not capturing variables by reference unless you have to, since when the request completes and the function executes, the variables can already be destructed. Advanced reference can be found [here](https://dpp.dev/lambdas-and-locals.html). Now let's see callback functions in the action:
+
+~~~~~~~~~~~~~~{.cpp}
+#include <dpp/dpp.h>
+
+const std::string BOT_TOKEN = "Token Was Here";
```

Comment:
Still not following other docs. [Please look here for more information](https://dpp.dev/slashcommands.html)

Reply:
But some examples still have const std::string

Reply:
The only page that is doing this is the [first bot page](https://dpp.dev/firstbot.html). The reason for this is because it's a direct comparison to the javascript version on the right. Every other page doesn't do that. Please look to change it to something like this:
```
int main()
{
    dpp::cluster bot("token");

    // ...
}
```

This way you don't need any variable.

Reply:
yes master

Reply:
const std::string BOT_TOKEN is no mor